README.md

```diff
@@ -279,16 +279,26 @@ superpowers:
 Configuration
 -------------
 
+### Configuration file
+
+Hub stores its configuration in a file named "hub", located in a directory
+determined in accordance with the 
+[XDG Base Directory Specification](http://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html).
```

Comment:
I don't think this section is at all necessary. We can just mention somewhere that hub respects XDG_CONFIG_HOME, and that should be enough for people familiar with XDG to understand what's going on. We can keep referring to `~/.config/hub` in the rest of the documentation as before.